### ISSUE TITLE
Sessions shared

### ISSUE THREAD

**Issue by Decme** [OtterBrowser/otter-browser#196]
I downloaded February's version a few days back for testing from http://sourceforge.net/projects/otter-browser/

I've just downloaded March's version separately (again from Sourceforge). I didn't delete Feb version or move it, just put the March version in a different directory. When I opened the March version, it automatically had the tab session from the Feb version.

I've seen this issue with webkit/blink browsers before. Had Chromium portable, SRWare Iron portable, Comodo Dragon (installed), and they were constantly leeching settings from each other. When I had SRWare Iron open and double clicked the Chromium shortcut, instead of Chromium opening, a new SRWare iron tab simply opened.

Is Otter designed to be installable by default on Windows, or portable by default? (personal preference is portable)

Basically what I'm getting at, is this (tab sessions & other settings leeching across different versions of a browser installed in separate directories) is VERY undesirable behavior. Thoughts on the matter?
